translations/base/v1/NTACorporateInfoV20240101.json

```diff
@@ -47,9 +47,9 @@
             "correct": {
                 "comment": {
                     "text": [
-                        " * `\"1\"` indicates the record was corrected.",
-                        " * `\"0\"` indicates the record was not corrected.",
-                        "Example: `\"0\"`"
+                        " * `1` indicates the record was corrected.",
+                        " * `0` indicates the record was not corrected.",
+                        "Example: `0`"
```

Comment:
Thank you for your contributions to this pull request. I've noticed that this PR includes changes related to the corporate information API. Since these changes are not directly related to our current focus on the bank information API, could you please create a separate pull request for the corporate information-related updates? This will help us maintain a clear and organized review process and ensure each API's modifications are clearly documented and easy to track.

Reply:
Thanks for the review. I reverted the changes about corporate information.